lib/core-generators/new/templates/views/pages/legal/privacy.ejs

```diff
@@ -27,7 +27,7 @@
     <h5>Cookies</h5>
     <p>Information related to your interactions with the Service may be collected and stored by us in small data files—commonly referred to as cookies—that are placed onto your device by our servers. We may use temporary session cookies, which are terminated when you close your current session, to store certain personal information you provide for purposes of administering that session and facilitating transactions requested by you during that session. We also may use persistent cookies, which remain saved on your device after your current session is closed, to store information related to your interactions with the Service to facilitate future sessions.  Most browsers automatically accept cookies, but you can usually modify your browser settings to decline cookies or to notify you when a cookie is being placed on your device. Each browser is different, but generally users can remove cookies by following directions provided in the browser’s “help” file. Please note that, if you reject cookies or disable cookies, some features of the Service may not function properly.</p>
     <h5>Collection of information from minors</h5>
-    <p>The Service is intended for a general audience and it is not intended for children under the age of 13.  Use of the Service is prohibited for anyone under the age of 13.   If you are between 13 and 18 years of age, you must secure authorization from your parents or legal guardian before using the Service.  Although the Service may contain information that may be of interest to children, the Service is not directed at children and we do not knowingly collect or solicit personal information from children under the age 13.</p>
+    <p>The Service is intended for a general audience and it is not intended for children under the age of 16.  Use of the Service is prohibited for anyone under the age of 16.   If you are between 16 and 18 years of age, you must secure authorization from your parents or legal guardian before using the Service.  Although the Service may contain information that may be of interest to children, the Service is not directed at children and we do not knowingly collect or solicit personal information from children under the age 16.</p>
```

Comment:
Note: We use 16 instead of 13 here to err on the side of caution (because 16 is the default in the EU, unless overridden by a member state, as of today's date.)

If you're in the US, I think you'd be ok with 13.  I'm also not a lawyer, so there's that.